src/generic-provider-views/Browser.js

```diff
@@ -12,41 +12,51 @@ module.exports = (props) => {
   }
 
   return html`
-    <div class="Browser">
-      <header>
-        <input
-          type="text"
-          class="Browser-search"
-          placeholder="Search Drive"
-          onkeyup=${props.filterQuery}
-          value=${props.filterInput}/>
+    <div class="Browser Browser-viewType--${props.defaultViewType}">
+      <header class="Browser-header">
```

Comment:
would it make sense to refactor this into a `Header` component? So it is easier to follow?